docker/README.md

```diff
@@ -1,14 +1,15 @@
 ### Building Redox using Docker images with the toolchain
 
-*All you need is git, make, qemu, fuse and docker. The method requires a non-privileged user able to run the `docker` command, which is usually achieved by adding the user to the `docker` group.*
+*All you need is rustup, git, make, qemu, fuse and docker. The method requires a non-privileged user able to run the `docker` command, which is usually achieved by adding the user to the `docker` group.*
```

Comment:
Why do we need rustup here? This is requirements for the host system, and we only use rust inside the image.